test/NuGet.Core.Tests/NuGet.Packaging.Test/RuntimeModelTests/RuntimeDescriptionTests.cs

```diff
@@ -45,7 +45,7 @@ public void Merge_PrefersRightImports()
     [Fact]
     public void Merge_UsesLeftImportsIfRightEmpty()
     {
-        RuntimeDescription l = new(runtimeIdentifier: "rid", Array.Empty<string>(),  Array.Empty<RuntimeDependencySet>());
+        RuntimeDescription l = new(runtimeIdentifier: "rid", Array.Empty<string>(), Array.Empty<RuntimeDependencySet>());
```

Comment:
You need to rebase your branch. This was already fixed: https://github.com/NuGet/NuGet.Client/pull/5336/files

Reply:
Thanks! Fixed.